lib/tasks/figgy_mms_ids.rake

```diff
@@ -0,0 +1,8 @@
+namespace :figgy_mms_ids do
+  desc 'Create a translation map for Traject of mms ids of records in Figgy'
+  task build_translation_map: :environment do
+    puts("Pulling mms_records report from Figgy and creating Traject translation map - it's a big report, so be patient!")
```

Comment:
How long does it typically take to fetch/process the report to reseed the cache?